integration-tests/src/main/resources/application.properties

```diff
@@ -1,2 +1,3 @@
 quarkus.openapi-generator.spec."petstore.json".base-package=org.acme.openapi
-quarkus.openapi-generator.spec."openweather.yaml".base-package=org.acme.openapi.weather
\ No newline at end of file
+# since the file name has a space, we use the URI representation of this space here to not break the properties file
+quarkus.openapi-generator.spec."open%20weather.yaml".base-package=org.acme.openapi.weather
```

Comment:
maybe we could allow "open weather.yaml" and encode for the user it as it's done for paths?

Reply:
@michalszynkiewicz I had problems adding properties with spaces. It won't get read correctly. That's why I kept the encoding here to show how end-users should do this.

Reply:
btw, we have a nice wrong message for them as well:

```
The config property quarkus.openapi-generator.spec."open%20weather.yaml".base-package is required but it could not be found in any config source
```